src/proxy/ClientBuild.php

```diff
@@ -71,41 +71,63 @@ public function init()
     }
     
     private $_buildConfig;
-    
+
     public function setBuildConfig(array $config)
     {
         $this->_buildConfig = $config;
-        
+
         foreach ($config['tables'] as $tableName => $tableConfig) {
             if (!empty($this->optionTable)) {
-                $skip = true;
-                
-                foreach ($this->optionTable as $useName) {
-                    if ($useName == $tableName || StringHelper::startsWithWildcard($tableName, $useName)) {
-                        $skip = false;
-                    }
-                }
-                
+
+                $skip = $this->skipTable($tableName);
```

Comment:
Variable $skip is not needed i assume but maybe rename the `skipTable()` function to `isSkipableTable($tableName)` and use it directly like

```php
if ($this->isSkipableTable($tableName)) {
    continue;
}
```